src/OrchardCore.Modules/OrchardCore.Resources/package.json

```diff
@@ -12,11 +12,15 @@
     "jquery-resizable-dom": "0.35.0",
     "jquery.easing": "1.4.1",
     "js-cookie": "2.2.1",
+    "monaco-editor": "^0.22.3",
```

Comment:
Please update to https://www.npmjs.com/package/monaco-editor/v/0.23.0 and version in docs/resources/libraries/Readme.md